README.md

```diff
@@ -20,18 +20,40 @@ js-libp2p-bootstrap
 ## Usage
 
 ```JavaScript
+const Libp2p = require('libp2p')
 const bootstrap = require('libp2p-bootstrap')
-
-const options = {
-  list: <List of Multiaddrs>
-  interval: 5000 // ms, default is 10s
+const TCP = require('libp2p-tcp')
+const { NOISE } = require('libp2p-noise')
+let options = {
+    modules: {
+        transport: [ TCP ],
+        peerDiscovery: [ bootstrap ],
+        encryption: [ NOISE ]
+    },
+    config: {
+        peerDiscovery: {
+            bootstrap: {
+                list: [ // a list of bootstrap peer multiaddrs to connect to on node startup
+                  '/ip4/127.0.0.1/tcp/63785/ipfs/QmWjz6xb8v9K4KnYEwP5Yk75k5mMBCehzWFLCvvQpYxF3d',
+                  "/dnsaddr/bootstrap.libp2p.io/ipfs/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN",
+                  "/dnsaddr/bootstrap.libp2p.io/ipfs/QmQCU2EcMqAqQPR2i9bChDtGNJchTbq5TbXJJ16u19uLTa" 
+                  ],
+                  interval: 5000 // default is 10 ms
+            }
+        }
+    }
 }
 
-const b = new bootstrap(options)
+async function start () {
+  let libp2p = await Libp2p.create(options)
+
+  libp2p.on('peer', function (peerInfo) {
+    //found a peer
```

Comment:
Should we add a console.log here so that users just copy pasting this example would actually see something happen? If so, perhaps we should change the bootstrap addresses in this usage example to reflect diable peers, so that this simply works without any change. What do you think?

If so, we can use the public addresses in https://github.com/libp2p/js-libp2p/blob/v0.30.0/examples/discovery-mechanisms/1.js#L11-L18

Reply:
@acolytec3 did you consider this? I feel this would be useful for someone who just wants to experiment with this quite fast

Reply:
Yep, sorry for the delay.  Got sidetracked in my edits.  See the latest version.  It should reflect both of these comments.